UsageCollectorWorkerService/Services/DataSender/SenderService.cs

```diff
@@ -0,0 +1,32 @@
+using UsageCollectorWorkerService.Models;
+using UsageCollectorWorkerService.Services.DataHolder;
+
+namespace UsageCollectorWorkerService.Services.DataSender;
+
+public class SenderService : ISenderService
+{
+    private readonly HttpClient _httpClient;
+    private readonly IDataHolderService _dataHolderService;
+    
+    public SenderService(HttpClient httpClient, IDataHolderService dataHolderService)
+    {
+        _dataHolderService = dataHolderService;
+        _httpClient = httpClient;
+    }
+
+    public async Task PostRequestAsync(RootSysResUsageValues rootSysResUsageValues)
+    {
+        using HttpRequestMessage request = new();
+        request.Method = HttpMethod.Post;
+        request.Content = _dataHolderService.CreateStringContentForPostRequest(rootSysResUsageValues);
+
+        try
+        {
+            await _httpClient.SendAsync(request);
+        }
+        catch (Exception ex)
+        {
+            
```

Comment:
Catch should not be empty. If you dont want the program to stop just log the value. Or if you don't want to log it for some reason, add comment here.